packages/theme-chalk/src/color-picker.scss

```diff
@@ -6,9 +6,9 @@
 $color-picker-size: () !default;
 $color-picker-size: map.merge(
   (
-    'large': 36px,
+    'large': 40px,
     'default': 32px,
-    'small': 28px,
+    'small': 24px,
   ),
   $color-picker-size
```

Comment:
**Question:** about the size, can we use a dedicated variable for all sizes? Since discrete sizes all around this project would be difficult to manage.

Reply:
I agree.

Reply:
Please check it again.